make RIOT_CI_BUILD=1 CC_NOCOLOR=1 --no-print-directory -C ./tests/thread_flags test
r
ssh -t rici@saclay.iot-lab.info 'socat - tcp:nrf52832mdk-1.saclay.iot-lab.info:20000'
�!���!���.H.H�o start test, r to print it is ready
Help: Press s to start test, r to print it is ready
Help: Press s to start test, r to print it is ready
Help: Press s to start test, r to print it is ready
Help: Press s to start test, r to print it is ready
Help: Press s to start test, r to print it is ready
Help: Press s to start test, r to print it is ready
Help: Press s to start test, r to print it is ready
Help: Press s to start test, r to print it is ready
START
main(): This is RIOT! (Version: 2020.07)
START
thread(): waiting for 0x1...
main(): setting flag 0x0001
thread(): received flags: 0x0001
thread(): waiting for 0x1 || 0x64...
main(): setting flag 0x0064
thread(): received flags: 0x0064
thread(): waiting for 0x2 && 0x4...
main(): setting flag 0x0001
main(): setting flag 0x0008
main(): setting flag 0x0002
main(): setting flag 0x0004
thread(): received flags: 0x0006
thread(): waiting for any flag, one by one
thread(): received flags: 0x0001
thread(): waiting for any flag, one by one
thread(): received flags: 0x0008
main: setting 100ms timeout...
main: timeout triggered. time passed: 100021us
main: setting 100ms timeout (using uint64)...
main: timeout triggered. time passed: 100021us
SUCCESS
r
r
r
r
Traceback (most recent call last):
  File "/home/runner/work/RIOT/RIOT/tests/thread_flags/tests/01-run.py", line 33, in <module>
    sys.exit(run(testfunc))
  File "/home/runner/work/RIOT/RIOT/dist/pythonlibs/testrunner/__init__.py", line 27, in run
    child = setup_child(timeout, env=os.environ,
  File "/home/runner/work/RIOT/RIOT/dist/pythonlibs/testrunner/spawn.py", line 88, in setup_child
    sync_child(child, env)
  File "/home/runner/work/RIOT/RIOT/dist/pythonlibs/testrunner/spawn.py", line 125, in sync_child
    utils.test_utils_interactive_sync(child,
  File "/home/runner/work/RIOT/RIOT/dist/pythonlibs/testrunner/utils.py", line 30, in test_utils_interactive_sync
    _test_utils_interactive_sync(child, retries, delay)
  File "/home/runner/work/RIOT/RIOT/dist/pythonlibs/testrunner/utils.py", line 22, in _test_utils_interactive_sync
    child.expect_exact(ready_exp, timeout=0)
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/pexpect/spawnbase.py", line 421, in expect_exact
    return exp.expect_loop(timeout)
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/pexpect/expect.py", line 181, in expect_loop
    return self.timeout(e)
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/pexpect/expect.py", line 144, in timeout
    raise exc
pexpect.exceptions.TIMEOUT: Timeout exceeded.
<pexpect.pty_spawn.spawn object at 0x7fe02b4b6d90>
command: /usr/bin/make
args: [b'/usr/bin/make', b'cleanterm']
buffer (last 100 chars): 'ESS\r\n'
before (last 100 chars): ': setting 100ms timeout (using uint64)...\r\nmain: timeout triggered. time passed: 100021us\r\nSUCCESS\r\n'
after: <class 'pexpect.exceptions.TIMEOUT'>
match: None
match_index: None
exitstatus: None
flag_eof: False
pid: 53504
child_fd: 5
closed: False
timeout: 10
delimiter: <class 'pexpect.exceptions.EOF'>
logfile: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
logfile_read: None
logfile_send: None
maxread: 2000
ignorecase: False
searchwindowsize: None
delaybeforesend: 0.05
delayafterclose: 0.1
delayafterterminate: 0.1
searcher: searcher_string:
    0: 'READY'
/home/runner/work/RIOT/RIOT/tests/thread_flags/../../Makefile.include:770: recipe for target 'test' failed
make: *** [test] Error 1

Return value: 2
